### PR TITLE
Keras LinMod Example files

### PR DESCRIPTION
**Pull request by stevefw419**
Josh, can you please review when you get a chance and comment?

### CODE REVIEW

File: examples/keras_linmod_example/ananconda-project.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    packages:
+      - python=3.10.9
+      - pandas
+      - numpy
+      - scikit-learn
+      - scipy
+      - keras
+      - tensorflow
+      - mlflow=2.0.1
+      - ae5-tools
```
Comment (joshburt) [inline examples/keras_linmod_example/ananconda-project.yml]: I think `ae5-tools` can be removed.  I don't believe you are leveraging it.

File: examples/keras_linmod_example/pykeraslinreg.py [hunk 2]
```diff
@@ -0,0 +1,488 @@
...
+msle = MeanSquaredLogarithmicError()
+model.compile(loss=msle, optimizer=Adam(lr=learning_rate), metrics=["msle"])
+              
+# train the model
+with mlflow.start_run() as run:
+  history = model.fit(x_train_trans, y_train.values, epochs=10, batch_size=64)
+  loss = history.history['loss']
+  print("Active run_id: {}".format(run.info.run_id))
+  signature = infer_signature(x_test_trans, model.predict(x_test_trans))
+  mlflow.tensorflow.log_model(model, "tflinreg", signature=signature)
```
Comment (joshburt) [inline examples/keras_linmod_example/pykeraslinreg.py:460]: You are leveraging tensorflow autologging.  It looks like the model will automatically get logged for you in this scenario: https://mlflow.org/docs/2.0.1/python_api/mlflow.tensorflow.html#mlflow.tensorflow.autolog

Comment (joshburt) [reply examples/keras_linmod_example/pykeraslinreg.py:460]: Sorry, I added this comment to the wrong line initially. ;)

Comment (joshburt) [reply examples/keras_linmod_example/pykeraslinreg.py:460]: Does logging it this way give you a better signature on the model?

File: examples/keras_linmod_example/pykeraslinreg.py [hunk 3]
```diff
@@ -0,0 +1,488 @@
...
+
+# Now let's create a keras Neural Networks model to mimic linear regression on this data.
+hidden_units1 = 160
+hidden_units2 = 480
+hidden_units3 = 256
+learning_rate = 256
+
+# Creating model using the Sequential in tensorflow
+def build_model_using_sequential():
+  model = Sequential([
```
Comment (joshburt) [inline examples/keras_linmod_example/pykeraslinreg.py:438]: I used to use this pattern to define my models up until this last year.  My last data science team showed me a functional declaration which I found to be very nice.  I have a sample over here if you're interested in seeing the differences between the two methods for building up these sequentials:
https://www.kaggle.com/code/joshuacburt/mayo-clinic-strip-ai-model-training

**Review by joshburt — COMMENTED** [review]
Want to add this to the list of projects in the repo README as well?

File: examples/keras_linmod_example/ananconda-project.yml (hunk 1 above)
Comment (stevefw419) [reply examples/keras_linmod_example/ananconda-project.yml]: Thanks! Will do.

File: examples/keras_linmod_example/pykeraslinreg.py (hunk 3 above)
Comment (stevefw419) [reply examples/keras_linmod_example/pykeraslinreg.py:438]: Thanks for pointing me to this! Will take a look...

File: examples/keras_linmod_example/pykeraslinreg.py (hunk 2 above)
Comment (stevefw419) [reply examples/keras_linmod_example/pykeraslinreg.py:460]: Yes, seems to give me better info when I look at the end result on the tracking server.

**Comment by stevefw419** [comment]
> Will update example list in README as well.

**Pull request merged**